custom_components/robonect/utils.py

```
@@ -17,6 +17,18 @@
 
 
+def parse_duration_seconds_to_minutes(payload: str) -> str:
+    """Parse a duration in seconds to minutes appending it with min."""
+
+    match = re.search(r"(\d+)", payload)
+    if match:
+        seconds = int(match.group(1))
+        minutes = round(seconds / 60)
+        return f"{minutes} min"
+    else:
+        return "unknown duration"
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)